src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@use "./blocks/variables.scss" as *;
+@use "./blocks/reset.scss";
```

Comment:
reset and variables are not blocks, place them in styles folder

Reply:
reset and variables moved to styles folder